src/include/txn.h

```diff
@@ -323,30 +325,25 @@ struct __wt_txn {
  */
 
 /* AUTOMATIC FLAG VALUE GENERATION START 0 */
-#define WT_TXN_AUTOCOMMIT 0x000001u
-#define WT_TXN_ERROR 0x000002u
-#define WT_TXN_HAS_ID 0x000004u
-#define WT_TXN_HAS_SNAPSHOT 0x000008u
-#define WT_TXN_HAS_TS_COMMIT 0x000010u
-#define WT_TXN_HAS_TS_DURABLE 0x000020u
-#define WT_TXN_HAS_TS_PREPARE 0x000040u
-#define WT_TXN_IGNORE_PREPARE 0x000080u
-#define WT_TXN_PREPARE 0x000100u
-#define WT_TXN_PREPARE_IGNORE_API_CHECK 0x000200u
-#define WT_TXN_READONLY 0x000400u
-#define WT_TXN_RUNNING 0x000800u
-#define WT_TXN_SHARED_TS_DURABLE 0x001000u
-#define WT_TXN_SHARED_TS_READ 0x002000u
-#define WT_TXN_SYNC_SET 0x004000u
-#define WT_TXN_TS_READ_BEFORE_OLDEST 0x008000u
-#define WT_TXN_TS_ROUND_PREPARED 0x010000u
-#define WT_TXN_TS_ROUND_READ 0x020000u
-#define WT_TXN_TS_WRITE_ALWAYS 0x040000u
-#define WT_TXN_TS_WRITE_MIXED_MODE 0x080000u
-#define WT_TXN_TS_WRITE_NEVER 0x100000u
-#define WT_TXN_TS_WRITE_ORDERED 0x200000u
-#define WT_TXN_UPDATE 0x400000u
-#define WT_TXN_VERB_TS_WRITE 0x800000u
+#define WT_TXN_AUTOCOMMIT 0x00001u
```

Comment:
Trying to understand what changed here? I see you removed the `WT_TXN_TS_WRITE*` lines. But why did removing those modify all the earlier flags (which have the same values as before)? That is unexpected.

Reply:
Right! Thanks. I wasn't seeing the one-less-zero. I thought I was missing some kind of whitespace change.